satellite/gracefulexit/endpoint_test.go

```diff
@@ -824,7 +824,7 @@ func TestExitDisabled(t *testing.T) {
 
 		// Process endpoint should return immediately if GE is disabled
 		response, err := processClient.Recv()
-		require.True(t, errs2.IsRPC(err, rpcstatus.Unimplemented))
+		require.Error(t, err)
```

Comment:
I would prefer to at least check `errs2.IsRPC(err, rpcstatus.Unimplemented) || errs2.IsRPC(err, rpcstatus.Unknown)`. I already have a branch for this

Reply:
Why is that your preference when no code depends on it?

Reply:
Because then there is no guarantee that it is an error related to the endpoint being unimplemented on the server side as opposed to some other arbitrary graceful exit error.

Reply:
But `errs2.IsRPC(err, rpcstatus.Unknown)` will be true for basically any error.

Reply:
Only 2 out of the ~30 places where rpc errors are returned in the graceful exit endpoint are `Unknown`

Reply:
This is the branch I made earlier today. I wouldn't mind approving this PR, but I'd prefer the changes in the otherone: https://github.com/storj/storj/pull/3455

Reply:
Ok so you're protecting against it successfully contacting the RPC endpoint and returning an error, rather than an arbitrary error due to some other failure. My comment was about how most errors (in general, like io.EOF or whatever) have an unknown RPC status.

I'm cool with #3455.